package.json

```
@@ -3,8 +3,21 @@
     "version": "3.1.2",
     "description": "Decode JWT tokens, mostly useful for browser applications.",
     "main": "build/jwt-decode.cjs.js",
-    "module": "build/jwt-decode.esm.js",
-    "types": "index.d.ts",
+    "module": "build/jwt-decode.mjs",
+    "types": "index.d.mts",
+    "exports": {
+        ".": {
+            "types": {
+                "require": "./index.d.ts",
+                "import": "./index.d.mts",
+                "default": "./index.d.mts"
+            },
+            "require": "./build/jwt-decode.cjs.js",
+            "import": "./build/jwt-decode.mjs",
+            "default": "./build/jwt-decode.mjs"
```

The default export should match the `type` field in `package.json`. The absence of `type`, means it’s `commonjs`.

Also the value that matches the default can be omitted.

```suggestion
            "types": {
                "import": "./index.d.mts",
                "default": "./index.d.ts"
            },
            "import": "./build/jwt-decode.mjs",
            "default": "./build/jwt-decode.cjs.js"
```